crates/libcontainer/src/process/seccomp_listener.rs

```diff
@@ -91,14 +91,17 @@ fn sync_seccomp_send_msg(listener_path: &Path, msg: &[u8], fd: i32) -> Result<()
     let iov = [IoSlice::new(msg)];
     let fds = [fd];
     let cmsgs = socket::ControlMessage::ScmRights(&fds);
-    socket::sendmsg::<UnixAddr>(socket, &iov, &[cmsgs], socket::MsgFlags::empty(), None).map_err(
-        |err| {
-            tracing::error!(?err, "failed to write container state to seccomp listener");
-            SeccompListenerError::UnixOther(err)
-        },
-    )?;
-    // The spec requires the listener socket to be closed immediately after sending.
-    let _ = unistd::close(socket);
```

Comment:
Why we can remove `close()`

Reply:
I think `OwnedFd` will automatically call `libc::close()` when it goes out of function body.

Reply:
> The spec requires the listener socket to be closed immediately after sending.

We need to close it as soon as possible.

Reply:
L106 is the return statement, so `sendmsg()` will be immediately followed by the drop of the socket. 
However, considering that the spec emphasizes this point, I believe it's necessary to explicitly close the socket. I plan to call `drop(socket)` on L105. Do you think that's okay?